### PR TITLE
enforce single quote

### PR DESCRIPTION
**Pull request by erwinmombay**
Closes https://github.com/ampproject/amphtml/issues/2148

@cramforce PTAL

### CODE REVIEW

**Comment by cramforce** [comment]
LGTM

File: extensions/amp-analytics/0.1/amp-analytics.js [hunk 1]
```diff
@@ -134,8 +134,8 @@ export class AmpAnalytics extends AMP.BaseElement {
       for (const replaceMapKey in this.config_['extraUrlParamsReplaceMap']) {
         if (++count > MAX_REPLACES) {
           console./*OK*/error(this.getName_(),
-           "More than " + MAX_REPLACES.toString() +
-           " extraUrlParamsReplaceMap rules aren't allowed; Skipping the rest"
+           'More than ' + MAX_REPLACES.toString() +
+           ' extraUrlParamsReplaceMap rules aren\'t allowed; Skipping the rest'
```
Comment (jridgewell) [inline extensions/amp-analytics/0.1/amp-analytics.js]: Can we use `avoid-escape`, too?

File: extensions/amp-font/0.1/test/test-fontloader.js [hunk 2]
```diff
@@ -22,19 +22,19 @@ import * as sinon from 'sinon';
 adopt(window);
 
 /** @private @const {string} */
-const FONT_FACE_ = "\
+const FONT_FACE_ = '\
```
Comment (jridgewell) [inline extensions/amp-font/0.1/test/test-fontloader.js]: Backticks here would avoid that awkward `\` newline.

File: extensions/amp-analytics/0.1/amp-analytics.js (hunk 1 above)
Comment (erwinmombay) [inline extensions/amp-analytics/0.1/amp-analytics.js]: :+1: sgtm

**Pull request merged**